aws_get.py

```diff
@@ -20,34 +16,17 @@
 
 def aws_get_script(dataset=None):
     """Return AWS get script."""
-
-    # query
-    es_url = app.conf["GRQ_ES_URL"]
+    grq_es = get_grq_es()
     index = app.conf["DATASET_ALIAS"]
-    #facetview_url = app.conf["GRQ_URL"]
-    print(('%s/%s/_search?search_type=scan&scroll=10m&size=100' % (es_url, index)))
-    logging.debug(
-        '%s/%s/_search?search_type=scan&scroll=10m&size=100' % (es_url, index))
-    print(json.dumps(dataset))
-    logging.debug(json.dumps(dataset))
-
-    r = requests.post('%s/%s/_search?search_type=scan&scroll=10m&size=100' %
-                      (es_url, index), json.dumps(dataset))
-    if r.status_code != 200:
-        print(("Failed to query ES. Got status code %d:\n%s" %
-               (r.status_code, json.dumps(r.json(), indent=2))))
-        logger.debug("Failed to query ES. Got status code %d:\n%s" %
-                     (r.status_code, json.dumps(r.json(), indent=2)))
-    r.raise_for_status()
-    logger.debug("result: %s" % pformat(r.json()))
+    logger.debug("Dataset: {}".format(json.dumps(dataset, indent=2)))
+    paged_result = grq_es.es.search(body=dataset, index=index, size=100, scroll="10m")
+    logger.debug("Paged Result: {}".format(json.dumps(paged_result, indent=2)))
 
-    scan_result = r.json()
-    count = scan_result['hits']['total']
-    scroll_id = scan_result['_scroll_id']
+    count = len(paged_result["hits"]["hits"])
```

Comment:
@mcayanan: should this still pull from `paged_results["hits"]["total"]` still? Otherwise `count` will be the page size.

Reply:
@pymonger good catch. I've updated it to `paged_results["hits"]["total"]["value"]` as that is the ES 7.1 way to get the count now. Check out the latest changes: 34de2a2

Reply:
LGTM

Reply:
LGTM 👍